src/api/spec/controllers/staging/staging_projects_controller_spec.rb

```diff
@@ -94,13 +94,13 @@
       it 'returns the staging_project name xml' do
         assert_select 'staging_project' do
           assert_select 'staged_requests', 1 do
-            assert_select 'entry', 3
+            assert_select 'request', 3
           end
           assert_select 'untracked_requests', 1 do
-            assert_select 'entry', 1
+            assert_select 'request', 1
           end
           assert_select 'requests_to_review', 1 do
-            assert_select 'entry', 2
+            assert_select 'request', 2
           end
           assert_select 'missing_reviews', 1 do
             assert_select 'entry', 1
```

Comment:
'request' for missing_reviews too - these are just requests as well:

```
  <missing_reviews count="1">
    <entry id="3525778" creator="zaz" state="new" package="zaz"/>
  </missing_reviews>
``` 

A bug in itself though - as it doesn't report what's missing. But it still shouldn't be entry

Reply:
> Perhaps you should have checks and missing_checks before we talk

No, I checked it too. My question was regarding your comment "That's only staged_requests".. what did you want to mean with that @coolo ?

Reply:
> A bug in itself though - as it doesn't report what's missing. But it still shouldn't be entry

should it be "review" or "request" ?

Reply:
It's a request with missing reviews.

Reply:
My preference would be 
``` 
  <missing_reviews count="1">
    <request id="3525778" creator="zaz" state="new" package="zaz">
       <missing_review by_project='games' by_package='zaz'/>
    </request>
  </missing_reviews>

Reply:
I prefer just `review`, since it's just a review. As `misssing_package` is just a package as @hennevogel suggested on #8542.. would that work for you @coolo ?

Reply:
As I said, I don't care either way as long as it's consistent.

I checked our code and we look at `package`, `request` and `by` in the current JSON format. `by` and `package` only to display, `request` to manipulate the priority of it